server/ent/ent.js

```diff
@@ -15,10 +15,13 @@ const LABEL_DIVIDER = '__'
 const makeIdent = speciesEntry =>
 	speciesEntry.name + LABEL_DIVIDER + speciesEntry.ident
 
+// This likely doesn't do anything at the time of writing (5/7/2018)
```

Comment:
Lol I remember writing this. So is the check for non-monophyly happening elsewhere now?

Reply:
Looks like it just happens here https://github.com/hybsearch/hybsearch/blob/3a27b4728c900a9aef706436f0fb56b28a6b5552/server/ent/ent.js#L85